tests/e2e/dsc_creation_test.go

```diff
@@ -393,8 +400,8 @@ func (tc *testContext) testUpdateDSCComponentEnabled() error {
 		return fmt.Errorf("error after retry %w", err)
 	}
 
-	// Sleep for 40 seconds to allow the operator to reconcile
-	time.Sleep(4 * tc.resourceRetryInterval)
+	// Sleep for 2 min to allow the operator to reconcile
+	time.Sleep(2 * tc.resourceUpdateTimeout)
```

Comment:
i agree with these timeout settings, it was very difficult to set the value, i had to test it several times from my local and uncertainty is how it will behavior in openshift-ci.